isis/src/base/objs/PvlToJSON/PvlToJSON.cpp

```diff
@@ -7,9 +7,11 @@ find files of those names at the top level of this repository. **/
 #include "PvlToJSON.h"
 
 #include <nlohmann/json.hpp>
+#include <QFile>
```

Comment:
This looks like it's not needed. Was it required previously but included from another header before?

Reply:
Verified, not needed.